docs/sources/_index.md

```diff
@@ -3,6 +3,199 @@ aliases:
   - /docs/grafana/v1.1/
   - /docs/grafana/v3.1/
   - guides/reference/admin/
+cascade:
+  # Until a better mechanism for arrangement is thought of, the following sequence should be ordered alphabetically by the `_target.path` value.
+  - _target: { path: /docs/grafana/** } # Default to every page having "Enterprise" and "Open source" labels.
+    labels:
+      products:
+        - enterprise
+        - oss
+
+  - _target: { path: /docs/grafana/** }
+    labels:
+      products:
+        - enterprise
+        - oss
+
+  - _target: { path: /docs/grafana/*/administration/** }
+    labels:
+      products:
+        - cloud
+        - enterprise
+        - oss
+
+  - _target: { path: /docs/grafana/*/administration/enterprise-licensing/** }
+    labels:
+      products:
+        - enterprise
+
+  - _target: { path: /docs/grafana/*/administration/organization-management/** }
+    labels:
+      products:
+        - enterprise
+        - oss
+
+  - _target: { path: /docs/grafana/*/administration/provisioning/** }
+    labels:
+      products:
+        - enterprise
+        - oss
+
+  - _target: { path: /docs/grafana/*/administration/recorded-queries/** }
+    labels:
+      products:
+        - cloud
+        - enterprise
+
+  - _target: { path: /docs/grafana/*/administration/roles-and-permissions/access-control/** }
+    labels:
+      products:
+        - cloud
+        - enterprise
+
+  - _target: { path: /docs/grafana/*/administration/stats-and-license/** }
+    labels:
+      products:
+        - enterprise
+        - oss
+
+  - _target: { path: /docs/grafana/*/alerting/** }
+    labels:
+      products:
+        - cloud
+        - enterprise
+        - oss
+
+  - _target: { path: /docs/grafana/*/breaking-changes/** }
+    labels:
+      products:
+        - cloud
+        - enterprise
+        - oss
```

Comment:
@jdbaldry - should we take this out since this page doesn't exist in the 9.5 docs?
```suggestion
```

Reply:
Yeah, I think for the sake of making this file less confusing for future users, let's remove it.